baker_wet_cleaning_application/scripts/map_handling_behavior.py

```diff
@@ -35,21 +32,22 @@ def returnToRobotStandardState(self):
 	def executeCustomBehavior(self):
 		
 		# Get the parameters for room sequencing
-		self.room_information_in_pixel_, self.segmented_map_ = self.database_handler_.getMapAndRoomInformationInPixel(self.rooms_list_)
+		(self.room_information_in_pixel_, self.segmented_map_) = self.database_handler_.getMapAndRoomInformationInPixel(self.rooms_list_)
 	
 		# Interruption opportunity
 		if self.handleInterrupt() >= 1:
 			return
 
 		# Room sequencing
-		self.room_sequencer_ = room_sequencing_behavior.RoomSequencingBehavior("Room sequencing", self.interrupt_var_, self.room_sequencing_service_str_)
```

Comment:
Please undo deleting the namespace everywhere. I want to see the namespace for knowing where things come from.

Reply:
Here it is clear, acceptable this way.